README.md

```diff
@@ -1,6 +1,7 @@
 # ExIRC
 
 [![Build Status](https://travis-ci.org/bitwalker/exirc.svg?branch=master)](https://travis-ci.org/bitwalker/exirc)
+![.github/workflows/tests.yaml](https://github.com/rockerBOO/exirc/workflows/.github/workflows/tests.yaml/badge.svg)
```

Comment:
Beware, you're using your own repo's status for this badge

Reply:
Good spot :) thanks.